docs/docs/scanner/vulnerability/os.md

```diff
@@ -9,25 +9,25 @@ To hide unfixed/unfixable vulnerabilities, you can use the `--ignore-unfixed` fl
-| Debian GNU/Linux                 | wheezy, jessie, stretch, buster, bullseye | Installed by apt/apt-get/dpkg |                 YES                  |
-| Ubuntu                           | All versions supported by Canonical       | Installed by apt/apt-get/dpkg |                 YES                  |
-| Distroless[^2]                   | Any                                       | Installed by apt/apt-get/dpkg |                 YES                  |
+| OS                               | Supported Versions                  | Target Packages               | Detection of unfixed vulnerabilities |
+|----------------------------------|-------------------------------------|-------------------------------|:------------------------------------:|
+| Alpine Linux                     | 2.2 - 2.7, 3.0 - 3.18, edge         | Installed by apk              |                  NO                  |
+| Wolfi Linux                      | (n/a)                               | Installed by apk              |                  NO                  |
+| Chainguard                       | (n/a)                               | Installed by apk              |                  NO                  |
+| Red Hat Universal Base Image[^1] | 7, 8, 9                             | Installed by yum/rpm          |                 YES                  |
+| Red Hat Enterprise Linux         | 6, 7, 8                             | Installed by yum/rpm          |                 YES                  |
+| CentOS                           | 6, 7, 8                             | Installed by yum/rpm          |                 YES                  |
+| AlmaLinux                        | 8, 9                                | Installed by yum/rpm          |                  NO                  |
+| Rocky Linux                      | 8, 9                                | Installed by yum/rpm          |                  NO                  |
+| Oracle Linux                     | 5, 6, 7, 8                          | Installed by yum/rpm          |                  NO                  |
+| CBL-Mariner                      | 1.0, 2.0                            | Installed by yum/rpm          |                 YES                  |
+| Amazon Linux                     | 1, 2, 2023                          | Installed by yum/rpm          |                  NO                  |
+| openSUSE Leap                    | 42, 15                              | Installed by zypper/rpm       |                  NO                  |
+| SUSE Enterprise Linux            | 11, 12, 15                          | Installed by zypper/rpm       |                  NO                  |
+| Photon OS                        | 1.0, 2.0, 3.0, 4.0                  | Installed by tdnf/yum/rpm     |                  NO                  |
+| Debian GNU/Linux                 | 7, 8, 9, 10, 11, 12                 | Installed by apt/apt-get/dpkg |                 YES                  |
```

Comment:
I felt it was worth switching from codenames to version numbers here, given that the table is getting rather wide now.